src/constructs/cloudwatch/ec2-alarms.ts

```diff
@@ -46,6 +51,34 @@ export class Gu5xxPercentageAlarm extends GuAlarm {
       alarmDescription: props.alarmDescription ?? defaultDescription,
       evaluationPeriods: props.numberOfMinutesAboveThresholdBeforeAlarm ?? 1,
     };
-    super(scope, id, alarmProps);
+    super(scope, AppIdentity.suffixText(props, "High5xxPercentageAlarm"), alarmProps);
+  }
+}
+
+/**
+ * Creates an alarm which is triggered whenever there have been several healthcheck failures within a single hour.
+ */
+export class GuUnhealthyHostsAlarm extends GuAlarm {
+  constructor(scope: GuStack, props: GuTargetGroupAlarmProps) {
+    const alarmName = `Unhealthy instances for ${props.app} in ${scope.stage}`;
+    const alarmDescription = `${props.app}'s instances have failed healthchecks several times over the last hour.
+      This typically results in the AutoScaling Group cycling instances and can lead to problems with deployment,
+      scaling or handling traffic spikes.
+
+      Check ${props.app}'s application logs or ssh onto an unhealthy instance in order to debug these problems.`;
+    const alarmProps = {
+      ...props,
+      alarmName: alarmName,
+      alarmDescription: alarmDescription,
+      metric: props.targetGroup.metricUnhealthyHostCount(),
+      treatMissingData: TreatMissingData.NOT_BREACHING,
+      statistic: Statistic.MAXIMUM,
+      threshold: 1,
+      comparisonOperator: ComparisonOperator.GREATER_THAN_OR_EQUAL_TO_THRESHOLD,
+      period: Duration.minutes(5),
+      datapointsToAlarm: 6,
```

Comment:
Based on the [docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html) (and confirmed via testing):

> When you configure Evaluation Periods and Datapoints to Alarm as different values, you're setting an "M out of N" alarm. Datapoints to Alarm is ("M") and Evaluation Periods is ("N").